content/en/deployment/strategies/blue-green.md

```diff
@@ -3,78 +3,87 @@ title: "Blue/Green Deployment Strategy"
-- (Optional) You can also create a `previewService` Kubernetes Service object so you can programmatically or manually observe the new version of your software before exposing it to traffic via the `activeService`. This is the `trafficManagement.kubernetes.previewService` field in the YAML configuration.
-
-## Add blue/green to your deployment
-
-1. In your deploy file, go to the `strategies` section.
-1. Create a new strategy named `blue-green-deploy-strat` like the following:
-
-   ```yaml
-   strategies:
-    blue-green-deploy-strat:
-      blueGreen:
-        redirectTrafficAfter:
-          - pause:
-              untilApproved: true
-        shutDownOldVersionAfter:
-          - pause:
-              untilApproved: true
-   ```
-
-   See the [Deployment File Reference]({{< ref "reference/deployment/config-file/strategies#bluegreen-fields" >}}) for an explanation of these fields.
-
-   This strategy is configured to pause for manual judgment before redirecting traffic to your new app version as well as before shutting down your old version. You could instead choose to pause for a duration of time.
-
-1. Change the value of `targets.<targetName>.strategy` for one or more of your deployment targets to `blue-green-deploy-strat`.
-
-   ```yaml
-   ...
-   targets:
-    <targetName>:
-      account: <agentIdentifier>
-      namespace: <namespace>
-      strategy: blue-green-deploy-strat
-    ...
-    ```
-1. At the bottom of your file, create a top-level traffic management configuration for your `activeService` and `previewService`:
-
-   ```yaml
-   ...
-   trafficManagement:
-     - targets: ['<targetName>']
-       kubernetes:
-         - activeService: myAppActiveService
-           previewService: myAppPreviewService
-   ...
-   ```
+For this guide, you need the following:
+
+- The [Armory CLI]({{< ref "cli.md" >}}) installed
+- A [Remote Network Agent]({{< ref "remote-network-agent/overview.md" >}}) (RNA) installed in the cluster where you want to deploy your app
+
+
+## Deploy your active service 
+You need to deploy a [Kubernetes Service object](https://kubernetes.io/docs/concepts/services-networking/service/) that sends traffic to the current version of your application. This is the `trafficManagement.kubernetes.activeService` field in the YAML configuration.
+
+As an example:
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: <your-app-service>
+  labels:
+    app: <your-app-name>
+spec:
+  selector:
+    app: <your-app-name>
+  ports:
+    - name: http
+      port: 80
+      targetPort: 9001
+      protocol: TCP
+```
+
+Save the file as `<your-app-service>.yaml` and run `kubectl -n <target-namespace> -f <your-app-service>.yaml` to deploy the service.
+
+## Create a blue/green deployment file
```

Comment:
I'd argue changing this section to call it defining Blue/Green strategy. 
because you don't need to make a new deployment file to use Blue/Green, you can use it just by defining it.

Reply:
we should break this down, declaring Blue/Green strat in the deployment config - this involves declaring strategy section and declaring traffic management section.

and then next is using Blue/Green strategy for an environment.